tests/ui/reify-intrinsic.stderr

```diff
@@ -23,9 +23,7 @@ LL |         std::intrinsics::unlikely,
    |
    = note: expected fn item `extern "rust-intrinsic" fn(_) -> _ {likely}`
               found fn item `extern "rust-intrinsic" fn(_) -> _ {unlikely}`
-   = note: different `fn` items always have unique types, even if their signatures are the same
-   = help: change the expected type to be function pointer `extern "rust-intrinsic" fn(bool) -> bool`
```

Comment:
Noting to myself: This suggestion is invalid.

Reply:
updated. the 'different `fn` items' note remains while the change suggestion is gone